CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+### Setup
```

Comment:
I would add an introduction, like

## Community contributions

We welcome community contributions. 

In order to facilitate an easier contribution and review process, we provide the following guidelines. Please review them before creating a pull request. 

This will improve the chances of your contribution being accepted, by facilitating a higher code quality, and help the maintainers to review the code as well.